functions.js

```diff
@@ -0,0 +1,134 @@
+//Task 1. Напишите функцию, которая принимает 2 числа и возвращает 1, если первое число больше, чем второе; 
+//-1, если первое число меньше, чем второе, и 0, если числа равны.
+//Попробуйте реализовать эту функцию не только с помощью if...else, 
+//но и с помощью тернарного оператора, и в виде стрелочной функции.
+// const num1 = +prompt('Enter a number');
+// const num2 = +prompt('Enter one more number');
+
+// function numberComparison(num1, num2) {
+//   if (num1 > num2) {
+//     return 1;
+//   }else if (num1 < num2) {
+//     return -1;
+//   }else if (isNaN(num1) || isNaN(num2)){
+//     return 'You entered a text. Enter a number, pls'
+//   }
+//     return 0;
+// };
+// alert(numberComparison(num1, num2));
+
+//function numberComparison(num1, num2) {
+//  (num1 > num2) ? alert('1'):
+//  (num1 < num2) ? alert('-1'):
+//  alert('0');
+//};
+//numberComparison(num1, num2);
+
+// const numberComparison = (num1, num2) => {
+//  if (num1 > num2) {
+//    return '1';
```

Comment:
why is it like text?

Reply:
i forgot that function can return a numbers